examples/logic_sig_example.js

```diff
@@ -1,6 +1,7 @@
 // Example: creating a LogicSig transaction signed by a program that never approves the transfer.
 
 const algosdk = require('../src');
+const { LogicSig } = require('../src/logicsig');
```

Comment:
nit: for these examples we shouldn't import any library code that wouldn't otherwise be available to normal dependencies, i.e. what `require('algosdk')` gives you.

You could use `algosdk.LogicSig` here instead of importing the class directly. (Or better yet, `algosdk.LogicSigAccount`, if that's what we prefer people use.)

Reply:
👍